Gemfile

```diff
@@ -92,6 +92,7 @@ gem "sprockets", "~> 3.7" # Sprockets is a Rack-based asset packaging system
 gem "staccato", "~> 0.5" # Ruby Google Analytics Measurement
 gem "storext", "~> 3.1" # Add type-casting and other features on top of ActiveRecord::Store.store_accessor
 gem "stripe", "~> 5.15" # Ruby library for the Stripe API
+gem "strong_migrations", "~> 0.6" # Catch unsafe migrations in development
```

Comment:
we should remove the development part from the comment but I'm also fine with this and we can clean up later

Reply:
So the impact of moving the `strong_migrations` gem in the Gemfile is so it's applied in any environment?

Reply:
yes